### ISSUE TITLE
NVDA intermittently loses access to the Outlook object model, making calendar unusable

### ISSUE THREAD

**Issue by leonardder** [nvaccess/nvda#11145]
### Steps to reproduce:
This issue occurs around half an hourafter I start my system. The first time I start Outlook, all is well, but after some time, NVDA is unable to get access to the object model.

### Actual behavior:
The following error is in de log:

```
ERROR - appModules.outlook.AppModule._get_nativeOm (09:29:57.766) - MainThread (2756):
Failed to get native object model
Traceback (most recent call last):
  File "comHelper.pyc", line 56, in getActiveObject
ValueError: invalid literal for int() with base 10: b''

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "appModules\outlook.pyc", line 120, in _get_nativeOm
  File "comHelper.pyc", line 58, in getActiveObject
RuntimeError: Helper process unable to get object; see log for details
```

Also, NVDA only says the word appointment when moving through appointments in the calendar. Information about unread state, attachments, is no longer read as well.

### Expected behavior:
NVDA should not lose access to the object model.

### Additional details
I'm pretty sure this started after I upgraded to Outlook 16.0.12730.20250

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-20175,9967903a

#### Windows version:
Windows 10 2004, 19041.208

#### Name and version of other software in use when reproducing the issue:
Outlook 16.0.12730.20250

#### Does the issue still occur after restarting your computer?
Not directly, but after some time (around half an hour). After that another restart is necessary.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, latest alpha's

#### If addons are disabled, is your problem still occuring?
Yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes

**Comment by leonardder** [comment]
Ugh, turns out I once changed my desktop icon to run NVDA as admin to test something, and never restored that. Three cheers for bad habits.
